package.json

```diff
@@ -214,6 +214,10 @@
                 "command": "aks.createClusterNavToAzurePortal",
                 "title": "Create Cluster From Azure Portal"
             },
+            {
+                "command": "aks.createCluster",
+                "title": "Create Cluster From VS Code"
```

Comment:
💡 purely for idea sake, shall we rename this to whatever the default configuration and sky it uses to create this test cluster. (AKA dev/test - or something ) I am not sure what is the default configuration eludes to.

Reply:
I like Create Cluster from VS Code :)

Reply:
Totally sounds good as name - I think I was keen to point out if either on webview or somehow we can be clear to state that - hey it creates some baseline as default - if and only if there is a way to summarise that in 3 to 4 words in a sentence. 💡 (so the key action is clearly conveyed as to what kind of default create it will do) 

will leave that as chain of thought here.

Reply:
I think adding something to the webview would be best, since the idea is that if we create more types/presets they will be selectable on the webview. I'll try to add something.

Reply:
<img width="472" alt="image" src="https://github.com/Azure/vscode-aks-tools/assets/1817696/d5d96f0b-e196-4b35-b1e5-d77382d3d65b">